GPT/gpt.py

```diff
@@ -133,8 +133,16 @@ def gpt_select_last():
         for _ in lines[0]:
             actions.edit.extend_left()
 
-    def gpt_apply_prompt(mode: str, prompt: str, source: str, destination: str):
+    def gpt_apply_prompt(
+        mode: str, prompt: str, source: str, destination: str
+    ) -> dict[str, str]:
         """Apply an arbitrary prompt to arbitrary text"""
+        response = actions.user.gpt_run_prompt(mode, prompt, source)
+        actions.user.gpt_insert_response(format_message(response), destination, mode)
```

Comment:
notably we are unwrapping and rewrapping the message here. I went down the path of trying to skip doing that and it tended up being a much bigger change, so I think this is better for now.

Reply:
Fine with me. My plan is to eventually have a GPTMessage class that makes this a bit easier to reason about.